Doc/reference/datamodel.rst

```diff
@@ -2849,6 +2849,47 @@ a :exc:`TypeError`.
       The specification for the Python ``match`` statement.
 
 
+.. _python-buffer-protocol:
+
+Emulating buffer types
+----------------------
+
+The :ref:`buffer protocol <bufferobjects>` provides a way for Python
+objects to expose efficient access a low-level memory array. This protocol
+is implemented by builtin types such as :class:`bytes` and :class:`memoryview`,
+and third-party libraries may define additional buffer types.
+
+While buffer types are usually implemented in C, it is also possible to
+implement the protocol in Python.
+
+.. method:: object.__buffer__(self, flags)
+
+   Called when a buffer is requested from ``self`` (for example, by the
+   :class:`memoryview` constructor). The ``flags`` argument is an integer
```

Comment:
arguments in italics: https://devguide.python.org/documentation/markup/#quick-reference

```suggestion
   Called when a buffer is requested from *self* (for example, by the
   :class:`memoryview` constructor). The *flags* argument is an integer
```

Reply:
Or remove *flags* as it's now private and meant to be undocumented:

https://github.com/python/cpython/pull/102521#issuecomment-1464730949

Reply:
This is the `flags` argument to `__buffer__`, not to memoryview.

(Also, thanks for the feedback! Will apply it later today.)